python/cuml/fil/fil.pyx

```diff
@@ -421,6 +446,23 @@ class ForestInference(Base):
                                                    algo, threshold,
                                                    storage_type)
 
+    @staticmethod
+    def load_from_skl(skl_model,
+                      output_class=False,
+                      threshold=0.50,
+                      algo='TREE_REORG',
+                      storage_type='DENSE',
+                      model_type="xgboost",
```

Comment:
awesome! would get rid of model_type param and call it `load_from_sklearn` but otherwise looks great.

Reply:
Done.